coralproject/talk
# Flag users as "Always premoderate"

## What does this PR do?
Issue #2030  
- Status can be set using the action dropdown in the People tab and in User Details.
- Users flagged as "Always premoderate" will have their comments sent to the premod queue.
- Users can be filtered with the Always Premoderate status.
- Include Always Premoderate status changes in User History.
- Show Always Premoderate status in Users CLI.
- Add spanish translations.

## How do I test this PR?
1. Go to the People tab and select Always Premoderate User from the actions dropdown (or from User Details)
2. Confirm in the dialog.
3. Log in as the premoderated user and write a comment.
4. Go to the Premod queue and the comment should be there.
